dump/src/error.rs

```diff
@@ -19,9 +19,9 @@ pub enum Error {
 impl ErrorCode for Error {
     fn error_code(&self) -> Code {
         match self {
-            // Are these three really Internal errors?
-            // TODO look at that later.
-            Error::Io(_) => Code::Internal,
+            Error::Io(e) => e.error_code(),
+
+            // These error come from an internal mis
```

Comment:
what's a "mis"?

Reply:
Oops, sorry, look like yesterday I left in the middle of writing a comment :grimacing: 
It's fixed now